packages/browser/lib/ui/form.ts

```diff
@@ -0,0 +1,83 @@
+import EventManager from "./eventManager";
+import { EvervaultFrame } from "./evervaultFrame";
+import type EvervaultClient from "../main";
+import type {
+  SelectorType,
+  EvervaultFrameHostMessages,
+  FormOptions,
+  FormPayload,
+  FormFrameClientMessages,
+} from "types";
+
+interface FormEvents {
+  ready: () => void;
+  error: () => void;
+  change: (payload: FormPayload) => void;
+  complete: (payload: FormPayload) => void;
```

Comment:
Here we have `complete` but bellow we dispatch `submitted` instead. We should change this to 

```
submitted: () => void
```